docker-compose.yml

```diff
@@ -1,4 +1,8 @@
 version: '3.6'
+
+volumes:
```

Comment:
??

Reply:
> ??

This command shares the volume between (possibly) several qiskit-service containers which is used by the SQLite database. This might be necessary because the 'transpile request' might block a single worker thread from other requests for quite a long time.